pkg/operator/controller/dns_status.go

```diff
@@ -17,6 +18,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+// conditionUpdateInterval is the update interval for progressing and degraded conditions.
+const conditionUpdateInterval = 20 * time.Second
```

Comment:
Is there a "why" for this interval? I see in the BZ that the lameduck mode for CoreDNS lasts 20s; is that what's represented by this value?

Reply:
> Exactly, that's the reason.

I think some commentary would help then. Even if it's just a reference to the BZ.

Reply:
I've added an explanation and a link to the BZ.